index.html

```diff
@@ -157,6 +160,31 @@ <h1 class="title">About me</h1>
                             </div>
                         </div>
                     </section>
+                    <section class = contact-section> 
+                        <h3 class ="contact-description">
+                            I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+                        </h3>
+                        <ul class="input-filed">
+                        <li><label for="email"> <input type="email" id="email" name="email" placeholder="Full name"></label></li>
+                       <li> <label for="e-mail">
+                        <input type="text" name="e-mail" id="e-mail" placeholder="e-mail address"></label></li>  
```

Comment:
- [ ] As it is in the requirement, all the input fields should have the `required` attribute, to ensure that proper information or data has been retrieved.